tests/conftest.py

```
@@ -0,0 +1,129 @@
+from random import choice, randint
+
+from pytest import fixture
+
+from can import Message
+from uds.addressing import AddressingType
+from uds.can import CanAddressingFormat, CanAddressingInformation, CanDlcHandler, CanSegmenter
```

Ruff (0.12.2)

`uds.can.CanAddressingInformation` imported but unused

Remove unused import: `uds.can.CanAddressingInformation`

(F401)